src/util/operations.tsx

```diff
@@ -13,3 +13,17 @@ export const getInstanceName = (operation: LxdOperation): string => {
       ?.split("?")[0] ?? ""
   );
 };
+
+export const getProjectName = (operation: LxdOperation): string => {
+  // the url can be
+  // /1.0/instances/<instance_name>?project=<project_name>
+  // when no project parameter is present, the project will be "default"
+
+  return (
+    operation.resources?.instances
+      ?.filter((item) => item.startsWith("/1.0/instances/"))
+      .map((item) => item.split("/")[4])
```

Comment:
Linking to non-default instances was broken -- I think because of this.
```suggestion
      .map((item) => item.split("/")[3])
```

Reply:
Thanks, totally missed this.